DESCRIPTION

```diff
@@ -1,38 +1,48 @@
 Package: rphylopic
-Title: Get 'Silhouettes' of 'Organisms' from 'Phylopic'
-Description: Work with 'Phylopic' web service (<http://phylopic.org/api/>) 
-    to get 'silhouette' images of 'organisms', search names, and more.
-    Includes functions for adding 'silhouettes' to both base plots and
-    'ggplot2' plots.
+Title: Get Silhouettes of Organisms from Phylopic
+Description: Work with the Phylopic Web Service (<http://api-docs.phylopic.org/v2/>) 
+    to fetch silhouette images of organisms, search names, and more.
+    Includes functions for adding silhouettes to both base plots and
+    ggplot2 plots.
 Version: 0.3.3.91
 Authors@R: c(
-    person("Scott", "Chamberlain", role = c("aut", "cre"), 
-        email = "myrmecocystus@gmail.com",
-        comment = c(ORCID = "0000-0003-1444-9135")),
-    person("David", "Miller", role = "ctb", 
-        email = "dave@ninepointeightone.net")
+    person(given = "William", family = "Gearty", email = "willgearty@gmail.com",
+      role = c("aut", "cre"), comment = c(ORCID = "0000-0003-0076-3262")),
+    person(given = "Lewis A.", family = "Jones", email = "LewisAlan.Jones@uvigo.es",
+      role = c("aut"), comment = c(ORCID = "0000-0003-3902-8986")),
+    person(given = "Scott", family = "Chamberlain", email = "myrmecocystus@gmail.com",
+      role = c("aut"), comment = c(ORCID = "0000-0003-1444-9135")),
+    person(given = "David", family = "Miller", email = "dave@ninepointeightone.net",
+      role = c("ctb"))
   )
-License: MIT + file LICENSE
-URL: https://github.com/sckott/rphylopic
-BugReports: https://github.com/sckott/rphylopic/issues
-Roxygen: list(markdown = TRUE)
+License: GPL (>= 3)
```

Comment:
curious why the change in license?

Reply:
Hey @sckott, this is the license we typically use for all our code (e.g. [palaeoverse](https://github.com/palaeoverse-community/palaeoverse)). As we've overhauled a lot of the package, we were planning to release it as a new major version, and it seemed an appropriate time to update the license to bring it in line with the other packages we maintain. Happy to discuss more if there is an issue?